### PR TITLE
chore: Remove unused Windows cse code

### PR DESCRIPTION
**Pull request by AbelHu**
Remove unused Windows cse code

### CODE REVIEW

File: parts/windows/kuberneteswindowssetup.ps1
Comment (mainred) [inline parts/windows/kuberneteswindowssetup.ps1:464]: The intention is to remove support of kubenet and else?
But IIRC, kubenet will be considered in the future, cc @keikhara

Comment (AbelHu) [reply parts/windows/kuberneteswindowssetup.ps1:464]: Remove all unused code for now. We do not know whether these code works with kubenet in aks. They are never validated in AKS. We should add kubenet support in cse when we plan to support it.

Comment (mainred) [reply parts/windows/kuberneteswindowssetup.ps1:464]: Please let me after you do a full validation under aks after removing the code, thanks.

Comment (AbelHu) [reply parts/windows/kuberneteswindowssetup.ps1:464]: I will wait until you approve it.

**Review by mainred — APPROVED** [review]

**Pull request merged**
